iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -309,9 +309,8 @@ public void commitAlterTable(org.apache.hadoop.hive.metastore.api.Table hmsTable
   }
 
   @Override
-  public void rollbackAlterTable(org.apache.hadoop.hive.metastore.api.Table hmsTable, EnvironmentContext context)
-      throws MetaException {
-    if (Boolean.valueOf(context.getProperties().getOrDefault(MIGRATE_HIVE_TO_ICEBERG, "false"))) {
+  public void rollbackAlterTable(org.apache.hadoop.hive.metastore.api.Table hmsTable, EnvironmentContext context) {
+    if (Boolean.parseBoolean(context.getProperties().getOrDefault(MIGRATE_HIVE_TO_ICEBERG, "false"))) {
```

Comment:
What is the difference here?
```
  public static Boolean valueOf(String var0) {
    return parseBoolean(var0) ? TRUE : FALSE;
  }
```

Reply:
Not much :) It was a checkstyle warning I think, that's why I made this change